composer.json

```diff
@@ -156,6 +157,10 @@
             },
             "drupal/entity_clone": {
                 "Issue #3036068: Make it possible to configure the published state of cloned entities.": "patches/entity_clone-3036068-20.patch"
+            },
+            "drupal/term_reference_tree": {
+                "Issue #2411667: \"Disabled\" state of the element is cleared.": "patches/term_reference_tree-2411667-disabled-state-of-element-is-cleared.patch",
+                "Issue #3251627: Add ability to enforce cascading selection..": "patches/term_reference_tree-3251627-enforce-selection.patch"
```

Comment:
Patches provide the functionality of disabling prisons when a category has been selected.